packages/react-components/src/Concepts/MigratingFromV8.stories.mdx

```diff
@@ -0,0 +1,182 @@
+
+It is recommended to depend on and import from `@fluentui/react-components`, rather than the individual packages (unless you have a specific reason to use an individual package). Depending on the suite helps ensure that you get consistent versions of all components and avoid duplicate dependencies. If you do not use all the components, most bundlers such as webpack and esbuild will automatically drop the unused code from your final bundle.
+
+Below is a table showing all components in the version 9 BETA release and in which package their code lives in:
+
+| Component        | Package in which it lives                                                                                 |
+| ---------------- | --------------------------------------------------------------------------------------------------------- |
+| `Accordion`      | [`@fluentui/react-accordion`](https://github.com/microsoft/fluentui/tree/master/packages/react-accordion) |
+| `Avatar`         | [`@fluentui/react-avatar`](https://github.com/microsoft/fluentui/tree/master/packages/react-avatar)       |
+| `Badge`          | [`@fluentui/react-badge`](https://github.com/microsoft/fluentui/tree/master/packages/react-badge)         |
+| `Button`         | [`@fluentui/react-button`](https://github.com/microsoft/fluentui/tree/master/packages/react-button)       |
+| `Divider`        | [`@fluentui/react-divider`](https://github.com/microsoft/fluentui/tree/master/packages/react-divider)     |
+| `FluentProvider` | [`@fluentui/react-provider`](https://github.com/microsoft/fluentui/tree/master/packages/react-provider)   |
+| `Link`           | [`@fluentui/react-link`](https://github.com/microsoft/fluentui/tree/master/packages/react-link)           |
+| `Menu`           | [`@fluentui/react-menu`](https://github.com/microsoft/fluentui/tree/master/packages/react-menu)           |
+| `MenuButton`     | [`@fluentui/react-button`](https://github.com/microsoft/fluentui/tree/master/packages/react-button)       |
+| `Popover`        | [`@fluentui/react-popover`](https://github.com/microsoft/fluentui/tree/master/packages/react-popover)     |
+| `Portal`         | [`@fluentui/react-portal`](https://github.com/microsoft/fluentui/tree/master/packages/react-portal)       |
+| `Text`           | [`@fluentui/react-text`](https://github.com/microsoft/fluentui/tree/master/packages/react-text)           |
+| `ToggleButton`   | [`@fluentui/react-button`](https://github.com/microsoft/fluentui/tree/master/packages/react-button)       |
+| `Tooltip`        | [`@fluentui/react-tooltip`](https://github.com/microsoft/fluentui/tree/master/packages/react-tooltip)     |
+
+---
+
+#### "Anything else I need to know about the file structure moving to the new version?"
+
+Well, there are actually a couple of things that have changed in version 9.
```

Comment:
I would propose that we include this kind of information in a `CONTRIBUTING.md` or a `README.md` for v9 and simply link here, this kind of information does not belong in a migration guide IMO

Reply:
I think I will leave this here until we create those markdown files to link to but I agree that once we do we might want to just put this over there and link from here.